content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/nodejs.md

```diff
@@ -51,7 +51,7 @@ If you are upgrading from a previous major version of the library (0.x, 1.x, 2.x
 
 ### Import and initialize the tracer
 
-Import and initialize the tracer either in code or with command line arguments. The Node.js tracing library needs to be imported and initialized **before** any other module.
+Import and initialize the tracer either in code or with command line arguments. The Node.js tracing library needs to be imported and initialized **before** any other module. Note that with frameworks like **Next.js** and **Nest.js** you must either provide an environment variable or add an additional Node.js flag. See [Complex framework usage][6] for more information.
```

Comment:
The only thing I'm worried about is reader skipped this sentence and going straight to [Option 1: Add the tracer in code](https://docs-staging.datadoghq.com/tlhunter/nodejs-link-complex-frameworks/tracing/trace_collection/automatic_instrumentation/dd_libraries/nodejs/#option-1-add-the-tracer-in-code)

Java has a tab setup right now which makes it easier to share links: https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/dd_libraries/java/?tab=wget#add-the-java-tracer-to-the-jvm, I wonder if this might be a way to improve visibility into these sections?
<img width="815" alt="image" src="https://github.com/user-attachments/assets/d83de98f-9820-4c73-9649-f4bf5e0cdd20">


Example tabs for NodeJS
- Using Code (which would have a warning about when to follow this)
- Using Command Line Arguments
- Using Bundling
- Using Esbuild

This is just a suggestion, I don't know if this would improve visibility 🤔

Reply:
I do think tabs would help consolidate the content on this page nicely. @tlhunter , I'd be happy to help make those tweaks in this PR or a separate one later if we just want to get the content out.

Alternatively, I think the intro content to **Import and initialize the tracer** is really text heavy. We could move some of it, like:
> After you have completed setup, if you are not receiving complete traces, including missing URL routes for web requests, or disconnected or missing spans, confirm the tracer has been imported and initialized correctly. The tracing library being initialized first is necessary for the tracer to properly patch all of the required libraries for automatic instrumentation.

further down the page, then put the new note in an alert banner to improve visibility.

Reply:
Chatted with @tlhunter offline. I'm going to merge the content so we can get it out there. Then, we'll do a follow up PR to organize into a tab structure.

As a short-term solution, I put the note into an alert banner to improve visibility in the meantime.